images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -96,4 +96,46 @@ Describe "GoogleCouldSDK" {
     It "gsutil" {
         "gsutil version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "Svn" {
+    It "svn exists" {
+        "svn --version --quiet" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Stack" {
+    It "Stack exists" {
```

Comment:
please, remove 'exists' from the simple tests like this, to be consistent with other tests